cruise-control/src/yaml/endpoints/load.yaml

```diff
@@ -0,0 +1,65 @@
+LoadEndpoint:
+  get:
+    operationId: load
+    summary: Get the cluster load once Cruise Control Load Monitor is running.
+    parameters:
+      - name: start
+        in: query
+        description: Start time of the cluster load
```

Comment:
For this line and every other description line - add period at the end please

Reply:
Done.